src/containers/ProfilePage/ToggleButton.jsx

```diff
@@ -1,23 +1,45 @@
 import React from "react";
 import { useAccordionToggle, AccordionContext } from "react-bootstrap";
+import i18n from "i18next";
 
 function getArrowClass(isCurrentElementSelected) {
-  return isCurrentElementSelected ? "arrow down" : "arrow right";
+  let arrow;
+
+  if (i18n.language === "ar") {
+    // if Arabic, make the margin from left
+    arrow = isCurrentElementSelected ? "arrow down-rtl" : "arrow left";
```

Comment:
return isCurrent ? x : y

same with line 12

Reply:
done!

Reply:
its not

Reply:
forgot to push, sorry @louisrli